features/bevragen/persoon-beperkt/adressering/adres-regels/fields-alias.feature

```diff
@@ -0,0 +1,178 @@
+#language: nl
+
+Functionaliteit: Persoon beperkt: adressering velden vragen met fields/fields alias
+
+  Wanneer een consumer van de BRP API niet is geautoriseerd voor het bevragen van de adresregel velden voor een verblijfplaats buitenland
+  Dan moet de consumer de adresseringBinnenland fields alias gebruiken om aan te geven dat hij adresregel velden voor een verblijfplaats binnenland vraagt
+
+Rule: de standaard adresregel (verkorte/volledige) field paden kan worden gebruikt door een consumer die is geautoriseerd voor het vragen van adresregels horende bij verblijfplaats buitenland
```

Comment:
suggereert misschien dat dit alleen gebruikt kan worden voor verblijfplaats buitenland. gaat om consumer die geautoriseerd is voor zowel binnenlands adres, binnenlandse locatie en verblijfplaats buitenland

Reply:
aangepast